src/wagtail_live/models.py

```diff
@@ -34,6 +34,8 @@ class LivePageMixin(models.Model):
     channel_id = models.CharField(
         help_text="Channel ID",
         max_length=255,
+        null=True,
```

Comment:
CharFields should (almost) never be Null. If Char is nullable, you'd have two values for empty. An empty string and Null. Read about it: https://docs.djangoproject.com/en/3.2/ref/models/fields/#null

I think this PR also needs a migration?

Reply:
Alright!

> I think this PR also needs a migration?

I don't think so, it's an abstract model. We don't have any migrations yet.